features/nanostack/mbed-mesh-api/source/include/mesh_system.h

```diff
@@ -28,7 +28,8 @@ extern "C" {
 enum {
     APPL_EVENT_CONNECT = 0x01,
     APPL_BACKHAUL_INTERFACE_PHY_DOWN,
-    APPL_BACKHAUL_INTERFACE_PHY_UP
```

Comment:
I don't fully understand why we don't distinguish APPL_BACKHAUL_INTERFACE_PHY_UP (interface swithced on, but cable not necessarily connected) and APPL_BACKHAUL_LINK_UP (cable connected)?

Reply:
@KariHaapalehto ?

Reply:
There haven't been need for that yet, from the connection staus point of view